### ISSUE TITLE
Error calling Pop on Windows

### ISSUE THREAD

**Issue by esDotDev** [lulupointu/vrouter#33]
I'm confused why this is throwing a runtime error when I call `VRouter.of(context).pop()`
![image](https://user-images.githubusercontent.com/736973/113469655-96799180-940c-11eb-82e3-44974f212ca9.png)

```
void main() {
  runApp(VRouter(
    initialUrl: HomePage.path,
    routes: [
      VNester(
          path: "/",
          widgetBuilder: (child) {
            return TabbedScaffold(body: child);
          },
          nestedRoutes: [
            VWidget(path: HomePage.path, widget: HomePage()),
            VWidget(path: SettingsPage.path, widget: SettingsPage()),
          ])
    ],
  ));
}

class TabbedScaffold extends StatelessWidget {
  const TabbedScaffold({Key? key, required this.body}) : super(key: key);
  final Widget body;

  @override
  Widget build(BuildContext context) {
    return Column(
      children: [
        Expanded(child: body),
        Row(
          children: [
            Expanded(
              child: TextButton(
                child: Text("Home"),
                onPressed: () => VRouter.of(context).push(HomePage.path),
              ),
            ),
            Expanded(
              child: TextButton(
                child: Text("Settings"),
                onPressed: () => VRouter.of(context).push(SettingsPage.path),
              ),
            ),
            Expanded(
              child: TextButton(child: Text("<< Back"), onPressed: () => VRouter.of(context).systemPop()),
            ),
          ],
        )
      ],
    );
  }
}

class HomePage extends StatelessWidget {
  static const path = "/home";
  @override
  Widget build(BuildContext context) {
    return Container(
      color: Colors.yellow.shade100,
      child: Text("HOME"),
    );
  }
}

class SettingsPage extends StatelessWidget {
  static const path = "/settings";
  @override
  Widget build(BuildContext context) {
    return Container(
      color: Colors.blue.shade100,
      child: Text("SETTINGS"),
    );
  }
}

```

**Comment by lulupointu** [comment]
Because of this: https://github.com/lulupointu/vrouter/blob/5d6af3cfffbcdaf10b7ee8558920ba72496b16ed/lib/src/vrouter/vrouter.dart#L1273

I did not update the check for desktop platform. I'll do it on Monday.

**Comment by esDotDev** [comment]
Nice, you might want to use this package, as `.io` namespace can't be accessed on web.
https://pub.dev/packages/universal_platform

**Comment by lulupointu** [comment]
Yes I know about it but I basically implemented my own universal platform. Plus I don't want to rely on another external package if I don't have to.

**Comment by kevlar700** [comment]
I believe there is another use or possibly abuse case as described by the flutter docs. Which is to use Navigator.pop() to close a drawer even on web when the screen is small.

I was about to try the following scaffold key technique?

"https://stackoverflow.com/questions/43807184/how-to-close-scaffolds-drawer-after-an-item-tap"

**Comment by lulupointu** [comment]
I know about the Drawer issue, I have given a fix in https://github.com/lulupointu/vrouter/issues/12#issuecomment-790422317

This is not perfect and `Navigator.pop` should ideally work but I would need to look a bit more into what is happening when the `Drawer` used. In the meantime the workaround is good enough I think. If this issue appears for other things I might spend more time into it.

**Comment by lulupointu** [comment]
Fixed in `VRouter 1.1.0+16`
